mobile/src/screens/Settings.tsx

```diff
@@ -0,0 +1,10 @@
+import React from 'react';
+import { Text, View } from 'react-native';
+
+export const Settings: React.FC = () => {
```

Comment:
I like the convention of ending each screen with `Screen`, can you add that here?